tools/integration-tests/tester/framework/docker.go

```diff
@@ -67,9 +68,14 @@ func (d *DockerContainer) CreateNode(ctx context.Context, conf config.GoShimmer)
 	}
 	cmd = append(cmd, conf.CreateFlags()...)
 
+	image := conf.Image
+	if image == "" {
+		image = "iotaledger/goshimmer"
+	}
+
```

Comment:
I think it would be better to define this when we create the configs (ie in `PeerConfig()`) and panic here if `conf.Image` is not set.